sky/provision/runpod/api/pods.py

```diff
@@ -0,0 +1,139 @@
+"""This module provides functions to generate GraphQL mutations for deploying
+spot instance Pods on RunPod.
```

Comment:
Please add the link for the original runpod python API for future reference : )

Reply:
added